### PR TITLE
Add 0.16.0 release notes to devel

### PR DESCRIPTION
**Pull request by dfarrell07**

### CODE REVIEW

**Review by tpantelis — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]

**Comment by dfarrell07** [comment]
> This branch cannot be rebased due to conflicts

???

There are no commit on devel and not on this branch.

> git log origin/devel ^merge_relnotes16

I guess we have some config that requires rebasing, not merging.

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -461,6 +521,8 @@ via the [Submariner addon](https://github.com/open-cluster-management/submariner
 * Submariner now supports SRV DNS queries for both ClusterIP and Headless Services. This facilitates Service discovery using
   port name and protocol. For a ClusterIP Service, this resolves to the port number and the domain name. For a Headless Service,
   the name resolves to multiple answers, one for each Pod backing the Service.
+* The CNI detection method in Submariner Operator is now improved to detect the Flannel CNI, even when the Flannel configMap
+  is missing from the cluster.
```
Comment (maayanf24) [inline src/content/community/releases/_index.en.md]: This is not related to 0.16.0 release notes, or did it got to the wrong place?

Comment (dfarrell07) [reply src/content/community/releases/_index.en.md]: @sridhargaddam sent this relnote to the 0.16 branch here: https://github.com/submariner-io/submariner-website/pull/1020

Comment (tpantelis) [reply src/content/community/releases/_index.en.md]: Apparently it was inadvertently put in the v0.10.1 section.

Comment (sridhargaddam) [reply src/content/community/releases/_index.en.md]: Oops, thanks for spotting it @maayanf24 and @dfarrell07.

Shall I send an updated PR to fix it in the release-notes-0.16 branch first?

Comment (sridhargaddam) [reply src/content/community/releases/_index.en.md]: Pushed a fix on the release-0.16 branch - https://github.com/submariner-io/submariner-website/pull/1059

**Review by sridhargaddam — APPROVED** [review]

**Review by skitt — APPROVED** [review]
